_topic_maps/_topic_map.yml

```
@@ -105,7 +105,7 @@ Topics:
   Distros: openshift-enterprise
 - Name: Admission plugins
   File: admission-plug-ins
-  Distros: openshift-enterprise,openshift-aro
+  Distros: openshift-enterprise,openshift-aro, openshift-origin
```

```suggestion
  Distros: openshift-enterprise,openshift-aro,openshift-origin
```
Doesn't look like there should be a space.

---

Whoops! Thank you!